horace_core/sqw/@gen_sqw_files_job/private/rundata_write_to_sqw_.m

```diff
@@ -44,6 +44,44 @@
 mpi_obj= MPI_State.instance();
 running_mpi = mpi_obj.is_deployed;
 
+% convert NaN values of runid to sensible values
+% if all values are NaN, assign values by file number
+% if some values are NaN, assign values greater than the maximum non-NaN
+% one
+
+% (a) - find largest non-nan run id
+%     - flag existence of any nans
+maxrunid = NaN;
```

Comment:
Defaulting this to `0` rather than `NaN` saves the extra work below with the observation about `max(0, NaN)`

Reply:
it doesn't give the extra clarity that using nan as a flag does